src/azure-cli/azure/cli/command_modules/network/_validators.py

```diff
@@ -1870,6 +1870,20 @@ def validate_status_code_ranges(namespace):
     namespace.status_code_ranges = values
 
 
+def validate_capture_size_and_limit(namespace):
+    if namespace.capture_limit:
```

Comment:
What was the current state due to which the limit and size functionality was broken ?

Reply:
There was no check for int type. It was also accepting strings and other values.